kcwitools/extract_optimal.py

```diff
@@ -0,0 +1,162 @@
+from __future__ import print_function
```

Comment:
best to use all the future pieces here.
See the other modules.

Reply:
Done!